Source/JavaWrappers.cpp

```diff
@@ -370,6 +406,27 @@ namespace java::net
     {
     }
 
+    bool URLConnection::GetDoOutput() const
+    {
+        auto output{m_env->CallBooleanMethod(JObject(), m_env->GetMethodID(m_class, "getDoOutput", "()Z"))};
```

Comment:
this coding style has evolved over time, please use this
```suggestion
        auto output = m_env->CallBooleanMethod(JObject(), m_env->GetMethodID(m_class, "getDoOutput", "()Z"));
```

I'm been telling people to follow the solution for this question: https://stackoverflow.com/questions/9976927/when-to-use-the-brace-enclosed-initializer/